<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsSchedule</class>
 <widget class="QWidget" name="SettingsSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>366</width>
        <height>549</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupXmltv">
         <property name="title">
          <string>XMLTV settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioFile">
            <property name="text">
             <string>Local file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;i&gt;XMLTV settings require restart.&lt;/i&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkRefresh">
            <property name="text">
             <string>Refresh XMLTV file on startup</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="BrowseWidget" name="location" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioNetwork">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Download file</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinOffset">
            <property name="suffix">
             <string> h</string>
            </property>
            <property name="minimum">
             <number>-12</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Offset:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>common/BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioNetwork</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
